compiler/damlc/daml-ide-core/test/Development/IDE/State/API/Testing.hs

```diff
@@ -89,6 +91,7 @@ data ShakeTestError
     | ExpectedVirtualResourceNote VirtualResource T.Text (Map VirtualResource T.Text)
     | ExpectedNoVirtualResourceNote VirtualResource (Map VirtualResource T.Text)
     | ExpectedNoErrors [D.FileDiagnostic]
+    | ExpectedNoMisMatch String String
```

Comment:
Maybe this is a bad idea or should I move the tests which use this to this module?

Reply:
`ExpectedNoMismatch` seems a bit too generator. How about `ExpectedTemplateProps [TemplateProps] [TemplateProps]` where the two lists correspond to actual and expected.